data/hai outfits.txt

```diff
@@ -247,7 +247,7 @@ outfit "Hai Williwaw Cooling"
 	"mass" 8
 	"outfit space" -8
 	"cooling" 9.5
-	description "While Hai technology is fairly heat efficient, occasionally they outfit their ships such that they sporadically overheat and shut down. When that happens, they install these."
+	description "While Hai technology is fairly heat-efficient, occasionally they outfit their ships such that they sporadically overheat and shut down. When that happens, they install these."
```

Comment:
It looks and sounds worse with no hyphen. The two hyphenated words are acting together as an adjective.